src/openslide-vendor-dicom.c

```diff
@@ -88,9 +88,13 @@ static const char *const ORIGINAL_TYPES[] = {
 static const char *const RESAMPLED_TYPES[] = {
   "DERIVED", "PRIMARY", "VOLUME", "RESAMPLED", NULL
 };
+static const char *const DERIVED_ORIGINAL_TYPES[] = {
```

Comment:
Nit: let's move this before `RESAMPLED_TYPES`, and maybe add a comment explaining when we'd see it.

Reply:
Done.